### PR TITLE
Fixed null values being displayed in Tasks Tab of User Info Screen.

### PR DESCRIPTION
**Pull request by ayushs136**
**What kind of change does this PR introduce?**
Fixed null values being displayed in Tasks Tab of User Info Screen. The screen is now getting data from database, which user has filled. 

**Issue Number:**
#409 

**Did you add tests for your code?**
No

**Snapshots/Videos:**
image

**Does this PR introduce a breaking change?**
Yeah

**Other information:**
Also Formatted date in dd-mm-yyyy format.

**Have you read the contributing guide?**
Yeah

### CODE REVIEW

**Comment by Sagar2366** [comment]
1. @ayushs136 seems like this issue is not yet fixed properly.
For example, check for test Test user users task list, it is throwing error.

2. Also, you have mentioned in PR description that this PR introduces braking change, can you please explain more on that ?

3. Please share video demo after adding changes for this fix.

Screenshot 1942-12-29 at 12 09 14 PM

**Comment by ayushs136** [comment]
@Sagar2366 Made the required changes. Please review.

**Comment by Sagar2366** [comment]
@ayushs136 please add unit test for your code.

**Comment by Sagar2366** [comment]
@ayushs136 can you please pull latest changes added to master for travis ci fix and send PR  again for review ?
Also disable code formatter and remove changes which are not required to fix the issue.

**Review by Sagar2366 — APPROVED** [review]

File: lib/views/pages/organization/switch_org_page.dart [hunk 1]
```diff
@@ -126,70 +120,73 @@ class _SwitchOrganizationState extends State<SwitchOrganization> {
...
-                itemIndex = userOrg[index]['_id'].toString();
-              });
-            },
-          );
-        },
-        separatorBuilder: (BuildContext context, int index) {
-          return Divider();
-        },
-      ),
+              padding: EdgeInsets.only(top: 10.0),
```
[the remark this answers is not in the document]
Comment (ayushs136) [reply lib/views/pages/organization/switch_org_page.dart:123]: Okay

[the remark this answers is not in the document]
Comment (ayushs136) [reply lib/views/pages/organization/switch_org_page.dart:123]: @Sagar2366 there was another issue in switch_org_page.dart, and it contains that changes, I missed to make a issue for that but you can find the screenshots attached to it here.

From
image

To
image

**Comment by ayushs136** [comment]
Fixed all issues and reverted unused changes. Please check out @Sagar2366! Meanwhile can I create another issue for the above switch_org_page.dart one?

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**